src/App.jsx

```diff
@@ -1,35 +1,88 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState, useEffect } from 'react';
+import axios from 'axios';
+
+import Result from './Result';
+import Game from './Game';
+import Welcome from './Welcome';
+
+import logo from './assets/logo.png';
 
 function App() {
-  const [count, setCount] = useState(0)
+  const [data, setData] = useState([]);
+  const [step, setStep] = useState(0);
+  const [correctAnswers, setCorrectAnswers] = useState(0);
+  const [start, setStart] = useState(true);
+  const [reset, setReset] = useState(true);
+
+  const onClickNext = () => {
+    setStep(step + 1);
+  };
+
+  const onClickAnswer = (answer) => {
+    if (data[step].correct_answer === answer) {
+      setCorrectAnswers(correctAnswers + 1);
+    }
+  };
+
+  const onStart = () => {
+    setStart(false);
+    setReset(false);
+  };
+
+  const onReset = () => {
+    setReset(true);
+    setStep(0);
+    setCorrectAnswers(0);
+    setStart(true);
+  };
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const res = await axios.get(
+          'https://opentdb.com/api.php?amount=10&category=9&difficulty=easy&type=multiple'
```

Comment:
Move API url to constants.

A good practice is to avoid any hardcoded strings in the `jsx` files